src/Elmah.Io.NLog/ElmahIoTarget.cs

```diff
@@ -63,11 +63,7 @@ protected override void Write(LogEventInfo logEvent)
 
         private string MachineName(LogEventInfo logEvent)
         {
-#if NET45
             return new MachineNameLayoutRenderer().Render(logEvent);
```

Comment:
If `MachineNameLayoutRenderer` works in both netstandard1.6 and net45, I don't see any need adding net46. The net45 build will work in .NET 4.6 as well. If I want to start using new features in .NET 4.6, I can add the framework later.